frontend/src/components/loginForm.tsx

```diff
@@ -58,6 +68,17 @@ export const LoginForm = () => {
               <div>
                   <button className='login_button' type="submit">Log in</button>
               </div>
+              <div className='orOAuth'>
+                <p>OR</p>
+              </div>
+              <div className='googleContainer' onClick={handleGoogleOAuth}>
+                <img src={googleIcon} alt='googleIcon'/>
+                <p>Log in with Google</p>
+              </div>
+              <div className='facebookContainer' onClick={handleFaceBookOAuth}>
+                <img src={facebookIcon} alt='googleIcon'/>
+                <p>Log in with Facebook</p>
+              </div>
```

Comment:
We're kind of using a div as a button. While that functionally works -- it's not a great practice.

If this is going to navigate us to a different URL we should probably be using links (`a`) but styled as a button

Reply:
fixed!